src/DelegateRegistry.sol

```diff
@@ -30,6 +30,10 @@ contract DelegateRegistry is IDelegateRegistry {
         amount
     }
 
+    /// @dev Standardizes vault address storage flags, can't use enums since we can't cast a value > type(enum).max to an enum type
+    address constant DELEGATION_UNKNOWN = address(0);
```

Comment:
Is unknown the right word, maybe null or empty

Reply:
The meaning we're trying to convey with the address(0) flag is that the delegation hash has not been seen before by the registry... so not sure about null or empty either